src/parity_testing/auth/testSuites/signOut/Test_that_globalSignOut_returns_partial_success_with_revoke_token_error.json

```diff
@@ -34,13 +34,14 @@
         {
             "type": "amplify",
             "apiName": "signOut",
-            "responseType": "complete",
+            "responseType": "success",
             "response": {
+                "signOutResult": "PARTIAL",
                 "globalSignOutError": null,
                 "hostedUIError": null,
                 "revokeTokenError": {
                     "exception": {
-                        "errorType": "RevokeTokenException",
```

Comment:
iOS doesn't have `RevokeTokenException`, so can we use `NotAuthorizedException` instead